packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -43,7 +44,13 @@ function useIsAccessibleToolbar( ref ) {
 
 	const determineIsAccessibleToolbar = useCallback( () => {
 		const tabbables = focus.tabbable.find( ref.current );
-		setIsAccessibleToolbar( hasOnlyToolbarItem( tabbables ) );
+		const onlyToolbarItem = hasOnlyToolbarItem( tabbables );
+		if ( ! onlyToolbarItem ) {
+			deprecated( 'Using custom components as toolbar controls', {
```

Comment:
Does this mean we should make the `__experimentalAccessibilityLabel` prop stable in this PR too, since that's the alternative right?

Reply:
Yes!